test/MUIDataTable.test.js

```diff
@@ -903,6 +902,88 @@ describe('<MUIDataTable />', function() {
     assert.equal(state.displayData.length, data.length);
   });
 
+  describe('should displayData consider filterOptions with logic', () => {
```

Comment:
Can we correct the English here? I think it should be `'should have the correct displayData and filters when using filterOptions with logic'`.

Reply:
Of cource! My English is very bad. Sorry!

Reply:
No worries! Thanks for working through this with me.